src/ui/EditTask.svelte

```diff
@@ -106,8 +107,15 @@
         parsedDone = parseDate('done', editableTask.doneDate);
     }
 
+
     onMount(() => {
+        const { globalFilter } = getSettings();
         const description = task.getDescriptionWithoutGlobalFilter();
+        // If we're displaying to the user the description without the global filter (i.e. it was removed in the method
+        // above), or if the description did not include a global filter in the first place, we'll add the global filter
+        // when saving the task
```

Comment:
Picking up a comment from lower down, I think it would be helpful if this comment could also say what happens if the global filter is empty.

Reply:
Done.